src/bundle.c

```diff
@@ -272,6 +272,11 @@ int remove_bundle(const char *bundle_name)
 		goto out_free_curl;
 	}
 
+	if (!search_bundle_in_manifest(current_mom, bundle_name)) {
+		printf("Bundle name is invalid, aborting removal\n");
```

Comment:
of course!

On Thu, Apr 7, 2016 at 3:37 PM, Patrick McCarty <notifications@github.com>
wrote:

> In src/bundle.c
> <https://github.com/clearlinux/swupd-client/pull/31#discussion_r58940554>:
>
> > @@ -272,6 +272,11 @@ int remove_bundle(const char *bundle_name)
> >  		goto out_free_curl;
> >  	}
> >
> > +	if (!search_bundle_in_manifest(current_mom, bundle_name)) {
> > +		printf("Bundle name is invalid, aborting removal\n");
>
> Since the "ret" variable is not set for this error condition, can we set
> ret to EBUNDLE_REMOVE in this case?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/clearlinux/swupd-client/pull/31/files/0ed3bbb5eaf37ed30e880ed7a383243a6a12860a#r58940554>
>